apps/src/studio/levels.js

```diff
@@ -2703,10 +2711,14 @@ levels.js_hoc2015_chain_characters = {
     {
       'id': 'playlab:js_hoc2015_chain_characters:calloutPlaceTwoWhenTauntaun',
       'element_id': '.droplet-gutter-line:nth-of-type(9)',
+      'hide_target_selector': '.droplet-drag-cover',
       'qtip_config': {
         'content' : {
           'text': msg.calloutPlaceTwoWhenTauntaun(),
         },
+        'hide': {
+          'event': 'mouseup touchend',
```

Comment:
can you remind me.. do we need both mouse events and a hide_target_selector to have a drag event dismiss a callout?  what's the logic here?